autoscale/src/main/java/com/sequenceiq/periscope/service/ScalingActivityService.java

```diff
@@ -103,6 +117,7 @@ public void update(Long activityId, FlowIdentifier flowIdentifier, ActivityStatu
         scalingActivity.setActivityStatus(activityStatus);
         scalingActivity.setScalingActivityReason(reason);
         LOGGER.info("Updating ScalingActivity: {} with FlowInformation and ActivityStatus: {}", activityId, activityStatus);
+        calculateUnprocessedScalingActivityMetrics();
```

Comment:
Performing this operation on every DB operation - would this get expensive ?
Instead could we have these metrics piggyback on one of the existing monitors, which runs more frequently - or even set up a separate Monitor / Timed Executor which updates these metrics ?

Reply:
Moved this to CleanupMonitor to run every 10 mins. I initially added this here because a similar pattern is followed in `ClusterService`, where metrics are calculated for every DB operation